docs/self-managed/zeebe-deployment/configuration/logging.md

```diff
@@ -3,65 +3,53 @@ id: logging
-      <Policies>
-        <TimeBasedTriggeringPolicy/>
-        <SizeBasedTriggeringPolicy size="250 MB"/>
-      </Policies>
-    </RollingFile>
-  </Appenders>
-
-  <Loggers>
-    <Logger name="io.camunda.zeebe" level="${env:ZEEBE_LOG_LEVEL:-info}"/>
-
-    <Logger name="io.atomix" level="${env:ATOMIX_LOG_LEVEL:-warn}"/>
-
-    <Root level="info">
-      <AppenderRef ref="RollingFile"/>
-
-      <!-- remove to disable console logging -->
-      <AppenderRef ref="${env:ZEEBE_LOG_APPENDER:-Console}"/>
-    </Root>
-  </Loggers>
-
-</Configuration>
-```
+[You can find the default log4j2.xml used by the application here](https://github.com/camunda/camunda/blob/main/dist/src/main/config/log4j2.xml).
+
+It will configure the default [log level](https://logging.apache.org/log4j/2.x/manual/customloglevels.html) to be `WARN` for all except for the
+following as `INFO`:
+
+- Camunda 8 (anything under `io.camunda` and `io.atomix`)
+- Spring and Spring Boot (anything under `org.springframework`)
+
+:::note
+[See our documentation on log levels to understand what the various log levels mean.](../../operational-guides/monitoring/log-levels.md)
+:::
+
+You can control the log level for the orchestration cluster via the `CAMUNDA_LOG_LEVEL` environment variable, e.g. `CAMUNDA_LOG_LEVEL=DEBUG`, or
+`CAMUNDA_LOG_LEVEL=ERROR`.
+
+:::note
+You can further use the following environment variables to set the log level for certain components of the application, but we recommend using the general
+`CAMUNDA_LOG_LEVEL` unless you really know what you're doing.
+
+- `ZEEBE_LOG_LEVEL`: will set the level for anything under `io.camunda.zeebe`, i.e. Zeebe related.
+- `ATOMIX_LOG_LEVEL`: will set the level for anything clustering or raft related.
+- `OPTIMIZE_LOG_LEVEL`: will set the level for anything under `io.camunda.optimize`.
+- `ES_LOG_LEVEL`: will set the level for anything under `org.elasticsearch`.
+  :::
+
+Additionally, it will configure three possible [appenders](https://logging.apache.org/log4j/2.x/manual/appenders.html) (aka outputs):
+
+- `RollingFile`: [A rolling file appender](https://logging.apache.org/log4j/2.x/manual/appenders/rolling-file.html) which prints out to a file
+  (by default, `logs/zeebe.log` relative from the distribution root). After a day, or once that file reaches 250MB, the file is "rolled" into a
+  compressed archive, and a new one is started. **This is enabled by default. You can disable it by setting the environment variable
+  `CAMUNDA_LOG_FILE_APPENDER_ENABLED=false`**.
+- `Console`: will output using the [Console Appender](https://logging.apache.org/log4j/2.x/manual/appenders.html#ConsoleAppender) and a
+  [pattern layout](https://logging.apache.org/log4j/2.x/manual/pattern-layout.html), directly to standard out. **This is enabled by default, and is
+  mutually exclusive with the `Stackdriver` appender. You can select one them via `ZEEBE_LOG_APPENDER`, e.g. `ZEEBE_LOG_APPENDER=Console` or
+  `ZEEBE_LOG_APPENDER=Stackdriver`.**
```

Comment:
🔧 I would first introduce, explain stackdriver and then talk about the mutually exclusivity, etc.